tox.ini

```diff
@@ -175,6 +175,7 @@ markers =
     baz
     # conftest.py reorders tests moving slow ones to the end of the list
     slow
+    uses_pexpect
```

Comment:
I would add a quick description here 👍

Reply:
Well, it is rather self-explaining, isn't it?

Reply:
Ok, done - also added a new "slowest_items" category.